psalm-baseline.xml

```diff
@@ -284,9 +284,18 @@
     </MixedArgumentTypeCoercion>
   </file>
   <file src="src/GridFS/CollectionWrapper.php">
+    <InvalidNullableReturnType>
+      <code><![CDATA[int]]></code>
+    </InvalidNullableReturnType>
     <MixedAssignment>
       <code><![CDATA[$ids[]]]></code>
     </MixedAssignment>
+    <NullableReturnStatement>
+      <code><![CDATA[$this->filesCollection->updateMany(
+            ['filename' => $filename],
+            ['$set' => ['filename' => $newFilename]],
+        )->getMatchedCount()]]></code>
+    </NullableReturnStatement>
```

Comment:
Boths additions will be removed in v2.x, thanks to https://github.com/mongodb/mongo-php-library/pull/1454